.rubocop.yml

```diff
@@ -656,6 +758,11 @@ Style/RedundantBegin:
 Style/RedundantException:
   Enabled: false
 
+Style/RedundantInitialize: # new in 1.27
+  Enabled: true
+  Exclude:
+    - 'spec/lib/presentable_spec.rb'
```

Comment:
...I'm not actually sure! the Presentable spec is terrifying and it might just be I didn't want to touch it, I can see at least one place this would be proccing that I cannot tell if is important